### DISCUSSION TITLE
Трудочасы на профессии

### DISCUSSION THREAD

**Discussion by Abirvalg228** [octothorp-team/public-dobrograd#919] [Предложения]
Предлагаю убрать искусственное, абсолютно не имеющее НИЧЕГО общего с ролевой игрой ограничение на получение профессий, которое состоит в необходимости наиграть миллионы часов для их получения. Это нелогично и просто отбивает желание у новых игроков вообще начинать игру. Почему я, будучи ролевиком с огромным стажем (на других серверах и платформах, в собственных крупных проектах) должен наигрывать 60 часов, чтобы стать мэром или 25 часов, чтобы стать, прости господи, медиком-копом? Это даже не 10 часов, за которые можно познать механики сервера, ознакомиться с картой и так далее и тому подобное. Это 60 часов, время на прохождение крупной сингл-плеер игры. Да я за все время в некоторые ММО , даже мои любимые, наиграл едва ли больше. На кого это рассчитано? На школьников, имеющих сотни часов, которые можно потратить на это или сервер все таки позиционирует себя как РП, а не гриндилка часов? Если очень хочется - введите хотя бы покупку часов через донат. Но в остальном - это издевательство. 

Аргумент про "это нужное время для понимания механик" - сразу мимо, т.к. за 250 рублей можно стать почти главным копом в участке, вообще не наигрывая часов, лол.

Предложение на замену: администраторы могут проверять уровень РП игрока сразу на месте, по расширенной его заявке и так далее.

**Comment by QixelPixel** [comment]
ПРОТИВ, Дооброград не похож на другие дарк рп сервера, тут и огромное кол-во разных механик, и более жесткие правила, 25 часов необходимо чтобы человек понял что это за сервер.
